collect_app/src/main/java/org/odk/collect/android/widgets/UrlWidget.java

```diff
@@ -43,18 +44,20 @@ public UrlWidget(Context context, QuestionDetails questionDetails, CustomTabHelp
 
     @Override
     protected View onCreateAnswerView(Context context, FormEntryPrompt prompt, int answerFontSize) {
-        binding = UrlWidgetAnswerBinding.inflate(((Activity) context).getLayoutInflater());
+        binding = WidgetAnswerBinding.inflate(((Activity) context).getLayoutInflater());
```

Comment:
I think this layout should be called `button_and_answer` or something. I don't like suggesting that it's the "default" answer view.

Reply:
I renamed it to `button_and_answer_layout`